TrainingExtensions/tensorflow/src/python/aimet_tensorflow/utils/common.py

```diff
@@ -55,6 +55,7 @@
     [["MaxPool"], ["MaxPool"]],
     [["AveragePool"], ["AvgPool"]],
     [["Relu"], ["Relu", "Relu6"]],
```

Comment:
Please also remove Relu6 from this line, I believe this is not the correct mapping and the one you have added is what we should have.